apps/microsoft-teams/frontend/src/customTypes/configPage.ts

```diff
@@ -1,4 +1,22 @@
-import { AppEventKey } from '@constants/configCopy';
+export const TOPIC_ACTION_MAP = {
+  'ContentManagement.Entry.create': 'created',
+  'ContentManagement.Entry.save': 'saved',
```

Comment:
I assume we'll just keep `save` and `auto_save` in here for now even though we aren't supporting those events in the UI currently?

Reply:
Ha ha no actually, this is a good catch.

I was originally thinking about aligning our types (and using the topic action map) but it was turning into too much trouble with too little benefit so I unwinded. But then I forgot to delete this constant. :)